tests/conftest.py

```diff
@@ -1,243 +1,257 @@
 import os
 from datetime import datetime
-from decimal import Decimal as PyDecimal
-import pytest
-from openpyxl import load_workbook
+from decimal import Decimal, DecimalException
+import pytest 
 from stock_indicators.indicators.common import Quote
+from dateutil.parser import parse, ParserError
 
 dir = os.path.dirname(__file__)
-data_path = os.path.join(dir, "../samples/quotes/History.xlsx")
-wb = load_workbook(data_path, data_only=True)
+
+def get_data_from_csv(sheet):
+    data_path = os.path.join(dir, f"../samples/quotes/{sheet}.csv")
+    with open(data_path ,"r") as f:
+        data = f.readlines()
+        data = [d.replace("\n","").split(",") for d in data] # parse csv
+    return data[1:] # skips the first row, those are headers
+
+def parse_decimal(value):
+    try:
+        return '{:f}'.format(Decimal(value))
+    except DecimalException:
+        return None
 
 @pytest.fixture(scope='session')
 def quotes(days: int = 502):
-    rows = list(wb['Default'])[1:]
+    rows = get_data_from_csv('Default')
 
     h = []
     for row in rows:
         h.append(Quote(
-            row[1].value,
-            row[2].value,
-            row[3].value,
-            row[4].value,
-            row[5].value,
-            row[6].value,
+            parse(row[1]),
+            row[2],
+            row[3],
+            row[4],
+            row[5],
+            row[6],
         ))
 
     h.reverse()
     return h[:days]
```

Comment:
This has "refactor" written all over it since it's just repeated code for every CSV file.  This should be a common method with `filename` and `days` as a parameters.

Reply:
there are varying number of headers in the csv files. while one has `i` instead of `index`, some do not have the index column. additionally, the headers are sometimes capitalised too.

Reply:
so essentially, the common method would need 3 parameters: `filename`, `days` and `containsIndex`.
where `containsIndex` implies that the csv file contains the index column.

could also name the argument `containsIndexColumn` to be more explicit.

Reply:
ugh.  that’s my fault.  disregard this one then.